.solargraph.yml

```diff
@@ -0,0 +1,22 @@
+---
+include:
+- "lib/**/*.rb"
```

Comment:
This will only index libraries, and won't index other files that could be helpful. Are you sure this is what you want with this PR?

Reply:
Great question. So far writing modules, just having that section helped a lot. The modules folder should definitely be excluded, but I'm unsure if there are any other folders that would be good to include.

Reply:
This is actually contradicted by the line below aka `**/*.rb` I think?

Reply:
It was there for redundancy. I ended up removing it after noticing no difference after checking my setup with and without the extra line

Reply:
The extra lines make intellisense function far better than it was before. Now lines have documentation as well as fairly accurate autocomplete options. One thing I notice is missing is intellisense on constants that would be used in module initialization. Module stability constants dont show up in autocomplete besides what ones vscode have seen before. On the bright side, vscode knows to lint with rubocop.